subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheDependencyResolutionFeaturesIntegrationTest.groovy

```diff
@@ -608,6 +608,24 @@ class ConfigurationCacheDependencyResolutionFeaturesIntegrationTest extends Abst
         new MavenLocalRepo() | _
     }
 
+    def "disables configuration cache when --export-keys is used"() {
```

Comment:
I'm not quite sure about this test is covering 100% what we want. Ideally it should export some keys, but configuration for exporting is rather complex, and considering this solution is short-term, I'm decided not to adding it. 
Opinions are welcome

Reply:
That's good enough.